package com.callisto.connect.error;

import java.util.HashMap;

public class ExceptionHelper {

	private boolean _isDebugging = true;
	
	public static final int ERROR_INVALID_PARAMETER = 1;
	
	private static HashMap<Integer, String> _dictionary = null;
	
	static {
		
		ExceptionHelper._dictionary = new HashMap<Integer, String>();
		
		// Initializations
	}
	
	public static String getErrorMessageByCode(int errorCode)
	{
		if (ExceptionHelper._dictionary.containsKey(errorCode) != true)
		{
			return null;
		}
		
		return ExceptionHelper._dictionary.get(errorCode);
	}
}
